presto-main/etc/catalog/hive.properties

```diff
@@ -7,3 +7,7 @@
 
 connector.name=hive-hadoop2
 hive.metastore.uri=thrift://localhost:9083
+
+# Uncomment below lines if dynamic discovery of HMS URI needs to be enabled
+# hive.metastore.discovery-type=dynamic
+# hive.metastore.discovery-uri={"url":"http://localhost:5436/discover","headers":{"Rpc-Service":"hms","Rpc-Caller":"presto"}}
```

Comment:
remoe this

Reply:
How do you suggest a sample config for dynamic discovery be specified? Should an update in README suffice?